src/ontoform/transformers/subcellular_location.py

```diff
@@ -0,0 +1,14 @@
+import gzip
+import zipfile
+from typing import BinaryIO
+
+from ontoform.util import SupportedFormats
+
+
+def transform(src: BinaryIO, dst: BinaryIO, format: SupportedFormats) -> None:
+    with zipfile.ZipFile(src, 'r') as zip_file:
+        for item in zip_file.filelist:
+            with zip_file.open(item) as file:
+                if item.filename == 'subcellular_location.tsv':
+                    with gzip.open(dst, 'wb') as gzip_file:
+                        gzip_file.write(file.read())
```

Comment:
```suggestion
    with zipfile.ZipFile("subcellular_location.tsv.zip") as zip_file:
        with zip_file.open("subcellular_location.tsv") as file:
            with gzip.open(dst, "wb") as gzip_file:
                gzip_file.write(file.read())

```

I think we can just specify the file we want from inside the zip straight away.

Reply:
just applied the changes